### ISSUE TITLE
react/require-render-return shouldn't warn if render method has multiple return statements

### ISSUE THREAD

**Issue by smashercosmo** [yannickcr/eslint-plugin-react#555]
Consider the following example. Seems like a valid case to me.

``` javascript
export default class Link extends Component {

  static propTypes = {
    condition: React.PropTypes.bool
  };

  renderOne() {
    return (
      <span>One</span>
    );
  }

 renderTwo() {
    return (
      <span>Two</span>
    );
  }

  render() {
    const {condition} = this.props;
    if (condition) return this.renderOne();
    else return this.renderTwo()
}

```

**Comment by ljharb** [comment]
Any reason you haven't made those two into SFCs, and returned them via jsx?

This kind of seems like a special case you would just disable with an inline eslint comment.

**Comment by smashercosmo** [comment]
mmm... what does SFC stands for? :)

**Comment by smashercosmo** [comment]
Aaaa... got it)

**Comment by smashercosmo** [comment]
Well, probably because I don't want to create a lot of files with simple components. It doesn't make sense to create a component for such a simple markup.

**Comment by ljharb** [comment]
You don't have to put them in separate files - but imo simple markup is when it makes the *most* sense :-)
